alfalfa_worker/lib/auto_name_enum.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class AutoName(Enum):
```

Comment:
can you document this?

Reply:
yeah. I'll add a docstring. The gist is that it changed the `auto()` method so that the generated values are just the name.